pushmsixscript/docker.d/init_worker.sh

```diff
@@ -36,7 +34,16 @@ case $ENV in
 esac
 
 case $COT_PRODUCT in
-  firefox) ;;
+  firefox)
+    export BETA_APPLICATION_ID=9NZW26FRNDLN
+    export RELEASE_APPLICATION_ID=9NZVDKPMR9RD
+    export TASKCLUSTER_SCOPE_PREFIX="project:releng:microsoftstore:"
+    ;;
+  thunderbird)
+    export BETA_APPLICATION_ID=ZZZZZTBDZZZZZ
```

Comment:
Still working on getting the store side of things setup, will fill in these values when they are available.